internal/http/controller.go

```diff
@@ -179,6 +180,11 @@ func NewController(dir string) (Controller, error) {
 				client.WithUsername(p.Username)
 				client.WithPassword(p.Password)
 				client.WithTimeout(time.Second * DefaultTimeoutSeconds)
+
+				if p.ShortExpiration > 0 {
+					client.WithShortExpiration(p.ShortExpiration)
+				}
+
```

Comment:
Sorry, not sure what I was thinking before. Default value of an int is 0, so this if statement is kinda pointless except to prevent setting it to less than 0, my fault.